### PR TITLE
Mostly fixes to the new lodge map.

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request

image
image
image
image

</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by Guidesu** [comment]
I added a few doors to places that needed it.
I added an extra door to the master office and I put another fax machine so hunters might use it on one of the shacks.
I fixed the lockers issue and the sharpening block and removed some of the starting materials aswell.
Removed the organ fab and gave some extra organs instead.
Replaced the gate with R-wall due to the nature of the location of being the first defense of the place.
Added a alchemy set at one of the shacks due to what me and Trilby intended to give hunters an archaic method of making chems.



image
image

**Comment by cdb-is-not-good** [comment]
Something actually does occur to me- didn't they have a proper surgery table on the last map?

**Comment by Guidesu** [comment]
image


> Something actually does occur to me- didn't they have a proper surgery table on the last map?

They had.

**Comment by Guidesu** [comment]
New:
image
Old:
image

**Comment by cdb-is-not-good** [comment]
also for the record I meant a guncase like ah- 
image

This. im p retty sure it can hold other guns as well

**Comment by Guidesu** [comment]
> also for the record I meant a guncase like ah- image
> 
> This. im p retty sure it can hold other guns as well

I think giving them a gun that was intended to HM get would be alright

**Comment by Trilbyspaceclone** [comment]
gate should be wood
star-striker should be back on the wall like it was
water in the river is also an annoying thing

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
the above commit

**Comment by Trilbyspaceclone** [comment]
I also dislike the xenobotney stuff

**Comment by Guidesu** [comment]
> I also dislike the xenobotney stuff

All done

**Comment by Guidesu** [comment]
Water is good the overlay is nice.

**Comment by Trilbyspaceclone** [comment]
theirs a locker of some kind on the herblist spawn in the medical bay area

**Comment by Guidesu** [comment]
> theirs a locker of some kind on the herblist spawn in the medical bay area

OPS did not sat that ksksksk thank you

**Comment by cdb-is-not-good** [comment]
> gate should be wood star-striker should be back on the wall like it was water in the river is also an annoying thing

Why?

**Comment by Trilbyspaceclone** [comment]
metal gate dosnt look good
Star striker dosnt have ammo you shouldnt even be taking it down, if you ruin it for the round to bad
water decals are performance heavy and dont look good/cover up and darken water

**Comment by cdb-is-not-good** [comment]
> metal gate dosnt look good Star striker dosnt have ammo you shouldnt even be taking it down, if you ruin it for the round to bad water decals are performance heavy and dont look good/cover up and darken water

If it's just there for display than surely it should just be in a display case :p

**Comment by cdb-is-not-good** [comment]
Also something that someone pointed out and I kind of agree- use the regular dirt/grass tiles rather than the..... Underbrush looking stuff

**Comment by Memedoktor** [comment]
personal request but could we get an agri fridge again so storing foodstuffs is manageable with all the various produce and eggs and the like

**Comment by Guidesu** [comment]
> personal request but could we get an agri fridge again so storing foodstuffs is manageable with all the various produce and eggs and the like

When I arrive home I will continue

**Review by Trilbyspaceclone — APPROVED** [review]

**Comment by TheSleepyCatmom** [comment]
Asking @cdb-is-not-good to actually review this in game to make sure it's not fucky-wucky.

Also make sure to halve the spawn amount of the scrap area, replace the 'paths' with actual dirt like we do in the colony and elsewhere, and replace the 'grass' with the grass tiles we use everywhere else too.

**Comment by cdb-is-not-good** [comment]
> Asking @cdb-is-not-good to actually review this in game to make sure it's not fucky-wucky.
> 
> Also make sure to halve the spawn amount of the scrap area, replace the 'paths' with actual dirt like we do in the colony and elsewhere, and replace the 'grass' with the grass tiles we use everywhere else too.

Will do a walkthrough once the requested changes are made.

**Comment by Guidesu** [comment]
Alright i will s tart to work it in a moment...
i LOST everything due to energy shortage

**Comment by Guidesu** [comment]
I so tired from the college ;-;

**Comment by Memedoktor** [comment]
Btw i think theres something wrong the implantable bolt lance in the Hunt master armory. i think they're acting like regular bolt lances instead of the implantable kind.

File: maps/_HunterLodge/map/_Hunting_Lodge.dmm [hunk 1]
```diff
@@ -2322,13 +2648,24 @@
...
 /obj/effect/window_lwall_spawn/reinforced,
 /turf/simulated/floor/asteroid/grass/colonial1,
 /area/nadezhda/outside/forest/hunting_lodge)
 "Aw" = (
 /turf/simulated/floor/wood/wild4,
 /area/nadezhda/outside/forest/hunting_lodge)
+"Ax" = (
+/obj/structure/curtain/open/privacy,
+/obj/machinery/smartfridge/seeds{
+	name = s"\improper Refrigerated Seeds Storage"
```
Comment (Trilbyspaceclone) [inline maps/_HunterLodge/map/_Hunting_Lodge.dmm:2665]:
```suggestion
	name = "\improper Refrigerated Seeds Storage"
```

**Comment by Guidesu** [comment]
going sleep
please leave a review <3

**Comment by Guidesu** [comment]
> I did a bit of a goof and used the old map for reference, kekw.
> 
> To reiterate my points that are still relevant
> 
> -Lodge goofy ahh -Many places have been outright downgraded from previous iterations (see below)
> 
> image -Barns and botany now require you to walk halfway across the lodge, then halfway back again (Bad design) image -Botany is goofy again because now they get access to arguably the most OP machines that allow them to make any chemical in the game, the electrolyser, as it allows them to turn any medicine or chemical into its prerequisite components. Which essentially means: Everything
> 
> image -Bar is incredibly goofy, doesn't even have enough seats for the full lodge slots.
> 
> image -No surgical table? But you give them organs? A?
> 
> Also lodgers have like a 1% chance to succeed in organ replacement since Kazkins old medical PR, even with 80+ bio, trust me.
> 
> image -Every outsider in 10,000KM is now rushing the lodge roundstart
> 
> image -Goofy and will cause later balance concerns (e.g lodge no longer having to trade for mats). Also encourages the lodge to go and mine for plasteel/etc as they can make more weapons from that.
> 
> image -Why is the huntmasters office bigger than most of the small 'departments' the lodge has? Why is it bigger than the actual medical bay?
> 
> image -Goofy goober gatehouse, you implement the metal walls but forget to make the doors have blast-doors round-start that start closed to prevent instant looting via doorhacking, or by just breaking the door

Aren't you perma banned from Sojourn?

**Comment by Guidesu** [comment]
And you are still using the older version so.
Dully iguinored

**Comment by Guidesu** [comment]
> > > 
> > 
> > 
> > Aren't you perma banned from Sojourn?
> 
> Yeah and? You still have downstream this PR will trickle to.
> 
> Afterall, Risikio was also banned but was allowed to contribute game-changing PRs with literally zero testing that had to undergo changes almost immediately.

sniff.
I don't care about risiko.
I will not take criticism from someone who does not play or has access to the server anymore.
Have a good night.

Even if you have your points I am not in the mood its 01:02 and to be fairly honest I don't want to shape or change the map to someone with a power gaming point of view that plus that are not even on the server anymore.

I am taking feedback from the community you are not in the community

**Comment by Guidesu** [comment]
Take with a grain of salt.
Feedbacks are okay. You had your points but I don't and nor will bend over to make changes unless that Trilby/CDB or anyone who wants wishes to change. Community opinions are always welcome though don't feel necessary to take feedback from a person again that is permabanned.

Now I going to sleep

**Comment by cdb-is-not-good** [comment]
Kay, review time. 

First off, the grass wasn't entirely fixed. Still looks notably better, but in areas like this it's just - not good. Contrast is important, and having entirely samey tilesets all side by side it looks meh.
image
image


Machine spawns aren't fixed either. If you're using random spawns with a chance to be zero, use less. 
image

Small issue with medical- that morgue tray is unusable. You can't stand in front of a tray and open it, and it opens onto the only tile from which you can access it. Other than that, looks fine.
image

farming area- gonna be real, it's a bit of a mess. Having all the mounds just in a pile along the wall both looks bad and means constantly stepping over them, which while...lacking gameplay effect, is still just...looks sloppy. dont do this.
image

image
Gate- generally looks fine. why do they have a hoverpod, though? Or a cyborg auto-repair station? Why? a recharger i can accept especially with all the talk of making synths have to recharge, but the rest seems...superfluous? also some of the walls are still steel(the window walls particularly), i do think we have wood low walls. 

image
why? Why did you put this here? The lodge doesnt need access to drone/circuit stuff.

image
Why?

image

Actually i stand corrected about med. 5 o negative? really? also 8 toolboxes in the storage area is a big "???"

Final conclusion; still needs work.

**Review by TheSleepyCatmom — CHANGES_REQUESTED** [review]
Removal of the xenobotany explicit equipment and requirement to change things as requested by @cdb-is-not-good . Fix the flooring to be uniform for grass.

**Review by Trilbyspaceclone — DISMISSED** [review]
remove the CBO's runtimes cardboard box
the stair spam looks not that good
please make the paths grass again
please remove the water decals form the water

**Comment by cdb-is-not-good** [comment]
new review let's go.

image
Why have that gap in the wall?

image
image
I think I already complained about having massive amounts of blood. 6 bags of O- might be way too many. Ditto on three whole tubes of nanopaste.

image
I'm assuming there's just a buncha 'loot' spawns on this one rack, but why?

image
???? why stack so many of these on top? why not use regular reinforced windows? I guess it works but you should probably move it to be just one barricade+one grill and maybe a full window over top. It's a little janky to stack them like this.

image
That's a lotta medkits.

image
Why is this even here? 

Still a few things that could use adjustment I M O but substantially better than before.

**Comment by Trilbyspaceclone** [comment]
Still needs liches approval as well

**Comment by TheSleepyCatmom** [comment]
I'll consider these issues minor. I don't know what Lich's github is but someone poke him for this.

**Comment by Medizeal22** [comment]
Okay. Loaded it up and took a few pictures of things that need to be considered. Firstly, thank you for putting this together as well as you have. I do have some overarching and some specific concerns I'll list out here.

1. 
image
image
Double beds in the rooms beyond the little isolating room. Also take note of the the stairs that lead into the room. The direction may be off but it doesn't look "right" with how the room is structured. Consider using a door instead for these parts or just a curtain. If there was 1 tile either east or west then it'd be more simple to simply put the stairs then the curtains, sequentially rather than on top of one another.

2. 
image
This solar panel APC room confuses me. It's either too big or that random solar in there just needs to be elsewhere. It doesn't make a lot of sense.

3. 
image
Consider using a different material / set up for this as the textures are clipping here, causing only 3/4ths of the window to actually take up the space expected.

4. 
image
For some reason this smashes out round-start. Please check same.

5.
image

Is this necessary? See rant below for rational.

6.
image

Flooring in this room is jarring and confusing. Please make it more uniform and friendly to look at.



--------------------------------------------

So I love the new look of the Hunter's Lodge, especially if it actually does bring people back to the Lodge. There are some BIG WARNINGS I would like to push out though about the overall "strategy" that a number of mappers have been pushing for with it that has been a hot-topic of mine for years.

When you're trying to influence players to come back to an area, bigger is NOT better. In fact, bigger kills off your population's interest twice as fast. The reason why is because something this big and spread out encourages people to isolate themselves from one another. Considering there will be around 5 hunters on the better side of average, this means that they're less likely to see one another and contributes to a feeling of the compound (which is what this is now, it's not really a lodge anymore) feeling dead. My recommendation is to cut back on the number of "isolation" rooms as a result. I am NOT saying to get rid of all of them, I am saying to consider reducing the number of them as to encourage players to see one another more often. Remember, the philosophy of the lodge is that they are NOT Nadezhda nor are they trying to REPLACE Nadezhda. They're a group of people who more or less choose to live a more naturalistic lifestyle and their resources and technology should reflect that. They're NOT super advanced nor are they to expected to be competitively equipped as Nadezhda. Resist the urge of making the lodge the "do anything without security" space.

Another thing I am noticing is that there are a number of "doors with curtains". This is an effective strategy if used in small numbers. It's used excessively here to the point it becomes a bit of an unnecessary annoyance. I recommend  you use one or the other, but not always both. This goes with stairs as well. Make sure the direction and use of them makes practical sense. They don't need to be in the threshold of every room as it does raise some questions about relative elevation that need not be brought up really.

I'm curious to see what the water textures will do to the server. I would caution against them to some degree and heed any warning that Trilby gives you about using them. On solo-play it seems fine but when you put it on the live server it may contribute to lag.

Finally, consider naming your doors. This will help people figure out where things are a bit more easily without having to add a bunch of unnecessary windows.

**Comment by Guidesu** [comment]
kavrick is working on the new lodge.
Which he is a way better mapper than me.

**Pull request closed without merging**